department-of-veterans-affairs/caseflow
# Formatting Assign Hearings page for AMA appeals

Connects #7981 

### Description
This PR updates the Assign Hearings page to include a tab for AMA appeals ready to be scheduled. There will be a followup PR that will include AMA appeals in the backend request.

### Testing Plan
1. Go to the Assign Hearings page
1. Confirm the AMA appeals tab is empty
1. Confirm the user is still able to schedule hearings from the legacy appeals tab
